src/main/groovy/liquibase/harness/stress/StressTest.groovy

```diff
@@ -0,0 +1,75 @@
+package liquibase.harness.stress
+
+import liquibase.Scope
+import liquibase.database.jvm.JdbcConnection
+import liquibase.harness.config.DatabaseUnderTest
+import liquibase.harness.config.TestConfig
+import liquibase.harness.util.rollback.RollbackStrategy
+import liquibase.ui.UIService
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import static liquibase.harness.util.TestUtils.chooseRollbackStrategy
+import static liquibase.harness.util.TestUtils.executeCommandScope
+
+class StressTest extends Specification {
```

Comment:
We can use @Ignore("some comment") intead of commenting code

Reply:
done